ts/data/case.ts

```diff
@@ -184,7 +184,29 @@ export const caseDefinition: TableDefinition<Case, Test> = {
       type: "number",
     },
   ],
-  filters: caseFilters,
+  filters: [
+    ...caseFilters,
+    {
+      title: "Started After",
+      key: "STARTED_AFTER",
+      type: "text",
+    },
+    {
+      title: "Started Before",
+      key: "STARTED_BEFORE",
+      type: "text",
+    },
+    {
+      title: "Completed After",
+      key: "COMPLETED_AFTER",
+      type: "text",
+    },
+    {
+      title: "Completed Before",
+      key: "COMPLETED_BEFORE",
+      type: "text",
+    },
```

Comment:
These should be fine to add to the shared `caseFilters` constant. Otherwise they should work on all of the tables on the detail pages, but they'll only appear as options on the Cases tab